### PR TITLE
Added UI for the application

### PR DESCRIPTION
**Pull request by AleksandroBoko**
User interface was added. Now user can choice different points in the menu and do some operations with doctors, patients, medicines, reports using console mode.

### CODE REVIEW

File: HospitalToday/UI/MenuHandler.cs [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+            {
+                var report = reports.GetItemById(id);
+                if (report != null)
+                {
+                    var doctor = doctors.GetItemById(report.DoctorId);
+                    var patient = patients.GetItemById(report.PatientId);
+                    if (doctor == null || patient == null)
+                    {
+                        Console.WriteLine($"{report.Id} {report.Date}");
+                        Console.WriteLine($"{doctor.FirstName} {doctor.LastName}");
```
Comment (NemtsovN) [inline HospitalToday/UI/MenuHandler.cs]: Let's create some extension method for doctor and patient to display this data as a single string

Comment (AleksandroBoko) [reply HospitalToday/UI/MenuHandler.cs]: Extension methods were added.

File: HospitalToday/UI/MenuBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+{
+    class MenuBuilder
+    {
+        public MenuBuilder()
+        {
+            menuBuilder = new StringBuilder();
+            menuData = new MenuData();
+        }
+
+        private StringBuilder menuBuilder;
```
Comment (NemtsovN) [inline HospitalToday/UI/MenuBuilder.cs]: This fields can be readonly (if you are not changing their objects later)

File: HospitalToday/UI/MenuBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+        public int MainMenu()
+        {
+            Clear();
+            menuData.MainMenuData.ForEach(x => menuBuilder.AppendLine(x));
+            Console.WriteLine(menuBuilder);
+            int req;
+            Console.WriteLine("Answer:");
+            var answer = Console.ReadLine();
+            var result = int.TryParse(answer, out req);
```
Comment (NemtsovN) [inline HospitalToday/UI/MenuBuilder.cs]: In general int is alias. It is better to use direct types for calling static members

File: HospitalToday/UI/MenuBuilder.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            return result ? req : -1;
+        }
+
+        public int DoctorMenu()
+        {
+            Clear();
+            menuData.DoctorsMenuData.ForEach(x => menuBuilder.AppendLine(x));
+            Console.WriteLine(menuBuilder);
+            int req;
+            Console.WriteLine("Answer:");
```
Comment (NemtsovN) [inline HospitalToday/UI/MenuBuilder.cs]: Minor suggestion:
it is better to separate lines by logic

File: HospitalToday/UI/MenuBuilder.cs (hunk 2 above)
Comment (AleksandroBoko) [reply HospitalToday/UI/MenuBuilder.cs]: Changed type of access - added key word readonly

File: HospitalToday/UI/MenuBuilder.cs (hunk 3 above)
Comment (AleksandroBoko) [reply HospitalToday/UI/MenuBuilder.cs]: I changed alias on the full type name.

File: HospitalToday/UI/MenuBuilder.cs (hunk 4 above)
Comment (AleksandroBoko) [reply HospitalToday/UI/MenuBuilder.cs]: Added indents to separate logic structure.

**Pull request merged**